### ISSUE TITLE
Generated vsix artefact doesn't contain the server jar

### ISSUE THREAD

**Issue by apupier** [camel-tooling/camel-lsp-client-vscode#14]
![image](https://user-images.githubusercontent.com/1105127/39118984-0cc40b3a-46eb-11e8-8f4c-dc2ac1a2d2f3.png)

only the 0.0.1 tag from the 19 March is correct:
![image](https://user-images.githubusercontent.com/1105127/39119001-18e1a97c-46eb-11e8-8f1d-00ae7584a64e.png)

**Comment by apupier** [comment]
the build contains this error:

```
node ./scripts/preinstall.js
(node:4098) UnhandledPromiseRejectionWarning: HTTPError: Response code 404 (Not Found)
    at EventEmitter.ee.on.res (/home/travis/build/camel-tooling/camel-lsp-client-vscode/node_modules/got/index.js:250:24)
    at emitOne (events.js:116:13)
    at EventEmitter.emit (events.js:211:7)
    at Immediate.setImmediate [as _onImmediate] (/home/travis/build/camel-tooling/camel-lsp-client-vscode/node_modules/got/index.js:99:8)
    at runCallback (timers.js:794:20)
    at tryOnImmediate (timers.js:752:5)
    at processImmediate [as _immediateCallback] (timers.js:729:5)
(node:4098) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:4098) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by apupier** [comment]
fixed by https://github.com/camel-tooling/camel-lsp-client-vscode/pull/16 but see end of discussion not sure to be a solution that will work on long-term

**Comment by lhein** [comment]
fixed by #28
